content/modules/redisai/redisai-quickstart.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: RedisAI Quick Start Tutorial
+description:
+weight: 20
+alwaysopen: false
+categories: ["Modules"]
+---
+For this quick start tutorial, we recommend that you use:
+
+- Either:
+    - [A Redis Enterprise Software database with the RedisAI module enabled]({{< relref "/modules/create-database-rs.md" >}})
```

Comment:
this should later on also point to the installation instructions

Reply:
I added a note to https://redislabs.atlassian.net/browse/DOC-254 to add a link to here.